arch/c4_context.md

```diff
@@ -0,0 +1,49 @@
+# Level 1: Radix System Context
+
+This page implement the first "C" in the [C4Model](https://c4model.com/#coreDiagrams): the **C**ontext Model.
+
+All production Systems (External and Radix) and Users that are commonly found on Radix networks are introduced on this page.
+
+![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
+
+# Shards and Chunks
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/), shards enable Radix Networks to scale infinitely.
+This is achieved by splitting up global state (Ledger) to different nodes. Each node can be part of one or more Shards slices - called Chunks.
+
+The minimal Shard space that a nodes can serve is one Chunk. The number of Chunks served depends on the hardware configuration (CPU, MEM, I/O) of
+a specific node. Each node has advanced algorithms for monitoring its own performance and dynamically - to other nodes non-deterministically - scale up and down
+ the number of chunks served.
+
+## Redundancy
+
+Nodes within the same Shard space are redundant to each other, which means that they serve and validate the same transactions.
```

Comment:
Only once a Node is synced up
Might be more accurate to say `Nodes with overlapping Shard chunks are redundant to each other within those chunks, which means ...`